.circleci/config.yml

```diff
@@ -45,6 +45,14 @@ jobs:
 
       - run: yarn lint-check
 
+      # - run: yarn link-check  # takes ~2min and fails
```

Comment:
?

Reply:
something to potentially only do for a daily/weekly cron job?
It takes too long and doesn't make sense to check all files (rather than just a diff as I do now) every single commit.

Reply:
Also it fails due to links which currently need fixing (which should be a different issue/PR)

Reply:
but command looks exactly the same as the previous one. Am I missing something here 🤔 ?

Reply:
ah, sorry ... lint and link .. omg

Reply:
I would say let's remove this for now .. circleci is not crone anyway, and I don't really like "dead"/"commented" code

Reply:
circleci can be easily configured to run cron jobs (about 5 extra lines of code in the config file) which I should probably add... What do you think?

Reply:
complete overhaul of circleci config (including daily full test) done now :confetti_ball: